plugin-localization/src/main/java/com/mapbox/mapboxsdk/plugins/localization/MapLocale.java

```diff
@@ -116,96 +121,103 @@
+   * Taiwan bounding Box extracted from OpenStreetMap
    */
   static final LatLngBounds TAIWAN_BBOX = new LatLngBounds.Builder()
           .include(new LatLng(26.389444, 118.115255566105))
           .include(new LatLng(21.733333, 122.107778)).build();
 
   /**
-   * Germany Bounding Box extracted from Open Street Map
+   * Germany bounding Box extracted from OpenStreetMap
    */
   static final LatLngBounds GERMANY_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(55.055637, 5.865639))
     .include(new LatLng(47.275776, 15.039889)).build();
 
   /**
-   * Korea Bounding Box extracted from Open Street Map
+   * Korea bounding Box extracted from OpenStreetMap
    */
   static final LatLngBounds KOREA_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(38.612446, 125.887108))
     .include(new LatLng(33.190945, 129.584671)).build();
 
   /**
-   * Japan Bounding Box extracted from Open Street Map
+   * Japan bounding Box extracted from OpenStreetMap
    */
   static final LatLngBounds JAPAN_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(45.52314, 122.93853))
     .include(new LatLng(24.249472, 145.820892)).build();
 
   /**
-   * France Bounding Box extracted from Open Street Map
+   * France bounding Box extracted from OpenStreetMap
    */
   static final LatLngBounds FRANCE_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(51.092804, -5.142222))
     .include(new LatLng(41.371582, 9.561556)).build();
 
   /**
-   * Russian Bounding box extracted from Open Street Map
+   * Russian bounding box extracted from OpenStreetMap
    */
   static final LatLngBounds RUSSIA_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(81.856903, -168.997849))
     .include(new LatLng(41.185902, 19.638861)).build();
 
   /**
-   * Spain Bounding box extracted from Open Street Map
+   * Spain bounding box extracted from OpenStreetMap
    */
   static final LatLngBounds SPAIN_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(27.4335426, -18.3936845))
     .include(new LatLng(43.9933088, 4.5918885)).build();
 
   /**
-   * Portugal Bounding box extracted from Open Street Map
+   * Portugal bounding box extracted from OpenStreetMap
    */
   static final LatLngBounds PORTUGAL_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(27.4335426, -18.3936845))
     .include(new LatLng(42.280468655, -6.3890876937)).build();
 
   /**
-   * Brazil Bounding box extracted from Open Street Map
+   * Brazil bounding box extracted from OpenStreetMap
    */
   static final LatLngBounds BRAZIL_BBOX = new LatLngBounds.Builder()
     .include(new LatLng(5.2842873, -33.8689056))
     .include(new LatLng(-28.6341164, -73.9830625)).build();
 
+  /**
+   * Vietnam bounding box extracted from OpenStreetMap
+   */
+  static final LatLngBounds VIETNAM_BBOX = new LatLngBounds.Builder()
+    .include(new LatLng(8.383333, 102.216667))
+    .include(new LatLng(23.666667, 109.466667)).build();
```

Comment:
Good point @1ec5, but in my opinion, this shouldn't be a huge issue since we're providing rough bounding boxes, not precise polygons. Additionally, using a point and a zoom level is pixel density-dependent and wouldn't be easy to hardcode as a constant.

How about we add the "Approximate" keyword to the Javadoc to not set the wrong expectations?

Reply:
I’d agree except that the Vietnam boundary is about half as wide as it would be if it included the disputed South China Sea Islands, while the China boundary extends well to the south to include them. The comments currently cite OpenStreetMap, but the bounding boxes don’t match OpenStreetMap either.

If there’s a technical constraint preventing us from using the same bounding boxes as the Mapbox Geocoding API or Mapbox Boundaries, then I think the comments should warn developers to convert the bounding boxes to points before use and shouldn’t mention OSM.

Reply:
Thanks for the feedback, you two. I'll touch up this pr once `8.3` is actually out.